spec/models/activity_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+
+RSpec.describe Activity, type: :model do
+  describe '.create' do
+    let!(:user) { create(:user, name: 'tester') }
+
+    context 'when create an activity' do
```

Comment:
hmm... no need to test the API? The creation is done by calling my API directly.